tensorflow_datasets/audio/groove.py

```diff
@@ -54,190 +54,191 @@
 _TIME_SIGNATURES = ["3-4", "4-4", "5-4", "5-8", "6-8"]
 
 _DOWNLOAD_URL = "https://storage.googleapis.com/magentadata/datasets/groove/groove-v1.0.0.zip"
-_DOWNLOAD_URL_MIDI_ONLY = "https://storage.googleapis.com/magentadata/datasets/groove/groove-v1.0.0-midionly.zip"
+_DOWNLOAD_URL_MIDI_ONLY = "https://storage.googleapis.com/magentadata/ \
+                          datasets/groove/groove-v1.0.0-midionly.zip"
 
 
 class GrooveConfig(tfds.core.BuilderConfig):
-  """BuilderConfig for Groove Dataset."""
-
-  def __init__(self, split_bars=None, include_audio=True, audio_rate=16000,
-               **kwargs):
-    """Constructs a GrooveConfig.
-
-    Args:
-      split_bars: int, number of bars to include per example using a sliding
-        window across the raw data, or will not split if None.
-      include_audio: bool, whether to include audio in the examples. If True,
-        examples with missing audio will be excluded.
-      audio_rate: int, sample rate to use for audio.
-      **kwargs: keyword arguments forwarded to super.
-    """
-    name_parts = [("%dbar" % split_bars) if split_bars else "full"]
-    if include_audio:
-      name_parts.append("%dhz" % audio_rate)
-    else:
-      name_parts.append("midionly")
-
-    v2 = tfds.core.Version(
-        "2.0.0", "New split API (https://tensorflow.org/datasets/splits)")
-    super(GrooveConfig, self).__init__(
-        name="-".join(name_parts), version=v2, **kwargs)
-    self.split_bars = split_bars
-    self.include_audio = include_audio
-    self.audio_rate = audio_rate
+    """BuilderConfig for Groove Dataset."""
+
+    def __init__(self, split_bars=None, include_audio=True, audio_rate=16000,
+                 **kwargs):
+        """Constructs a GrooveConfig.
```

Comment:
TF uses 2 spaces not 4.

Reply:
If I use 2 spaces I get these bad indentation errors:
```groove.py:62:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
groove.py:64:0: W0311: Bad indentation. Found 2 spaces, expected 4 (bad-indentation)
groove.py:65:0: C0330: Wrong continued indentation (remove 1 space).
                **kwargs):
               |^ (bad-continuation)
groove.py:66:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:76:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:77:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:78:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)
groove.py:79:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:80:0: W0311: Bad indentation. Found 10 spaces, expected 12 (bad-indentation)
groove.py:82:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:84:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:86:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:87:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)
groove.py:88:0: W0311: Bad indentation. Found 6 spaces, expected 8 (bad-indentation)```